tests/test_dependencies.py

```diff
@@ -0,0 +1,87 @@
+import pytest
```

Comment:
You wrote a lot of tests, but still not testing the ONLY function which should be tested: show_external_dependencies